lib/components/T3CeHeader/T3CeHeader.vue

```diff
@@ -2,16 +2,16 @@
   <div
     v-if="header && headerLayout !== 100"
     :class="headerCss"
```

Comment:
This dynamic class also should return the class name with the `t3` prefix. I also suggest to change name of this computed to headerClass